### DISCUSSION TITLE
Animal Status Criteria Proposal

### DISCUSSION THREAD

**Discussion by Magnus8377** [janstehno/cotw-companion#59] [Ideas]
I have an idea: Let's display the required class, score, and weight for an animal to achieve "diamond" or "great one" status.

**Comment by janstehno** [comment]
What exactly would you have in mind?

There already is a score for diamond. Also maximum weight which is shown in the in-game estimate weight range that could make a diamond. For a lot of animals weight does not matter. Difficulty matters in a small sample of cases where animal with one difficulty level less than its max could make a diamond.

Weight and score does not matter for great one, the difficulty simply has to be 10.

I do not think this in-game mechanic is deterministic enough and equal across all animals so that it could be shown as you would like to.

For now, not planned, unless I am convinced otherwise.

**Comment by 100luz-Max** [comment]
A good place to show the Diamond Trophy Score (and maybe also max difficulty and max weight) of the animals would be in the Reserve Screen, where all the animals are listed. Currently We can see the full list and if they have a great one.

So whenever you are hunting in a reserve, you can have all that information at a glance.
Attaching an example:

<img width="800" height="1600" alt="65bae99b-40a7-4613-9f8e-b121330e7efc" src="https://github.com/user-attachments/assets/27bad9bd-f367-4261-8264-2654def75b8c" />
